language/move-prover/bytecode-to-boogie/src/cli.rs

```diff
@@ -17,7 +17,11 @@ use std::sync::atomic::{AtomicBool, Ordering};
 pub const INLINE_PRELUDE: &str = "<inline-prelude>";
 
 /// Default flags passed to boogie. Additional flags will be added to this via the -B option.
-const DEFAULT_BOOGIE_FLAGS: &[&str] = &["-doModSetAnalysis", "-noinfer"];
+const DEFAULT_BOOGIE_FLAGS: &[&str] = &[
+    "-doModSetAnalysis",
+    "-noinfer",
+    "-printVerifiedProceduresCount:0",
```

Comment:
With this flag, Boogie wouldn't print out the number of functions verified. This number would confuse the users since it also includes native functions.